types/screeps/index.d.ts

```diff
@@ -4143,7 +4577,41 @@ interface SpawnOptions {
     directions?: DirectionConstant[];
 }
 
-interface SpawningConstructor extends _Constructor<Spawning>, _ConstructorById<Spawning> { }
+interface SpawningConstructor extends _Constructor<Spawning>, _ConstructorById<Spawning> {}
+interface StoreBase<POSSIBLE_RESSOURCES extends ResourceConstant, UNLIMITED_STORE extends boolean> {
```

Comment:
Not sure it's important, but here is a typo in POSSIBLE_RES~S~OURCES

Reply:
@scaytrase Thanks. It doesn't seem critical, it's used consistently and would only be referenced by users if they're extending the interface and referencing the type variable (rarely expected). I'll take a look at updating it in a patch release.

Reply:
Typo fixed in latest commit.